base/applications/taskmgr/taskmgr.h

```diff
@@ -15,7 +15,7 @@
 
 #define STATUS_WINDOW	2001
 #define STATUS_SIZE1	85
-#define STATUS_SIZE2	190
+#define STATUS_SIZE2	157 // he-IL.rc determines 72 requirement
```

Comment:
What do you mean by "72 requirement" ?

Reply:
I mean the minimum needed width of the size2 field. 157 - 85 == 72.

Reply:
Then this comment sentence needs to be clarified.

Reply:
Do you have a specific suggestion? I could also just strip it.

Reply:
Resolved